tests/e2e/specs/merchant/merchant-a-onboarding-task-list.spec.js

```diff
@@ -1,77 +1,10 @@
 /**
  * External dependencies
  */
-const { merchant, withRestApi } = require( '@woocommerce/e2e-utils' );
+const {
+	testAdminHomescreenTasklist,
+} = require( '@woocommerce/admin-e2e-tests' );
 
 describe( 'Onboarding > WooCommerce Setup Wizard & Task List', () => {
-	beforeAll( async () => {
-		await merchant.login();
-		await withRestApi.deleteAllShippingZones();
-	} );
-
-	afterAll( async () => {
-		await merchant.logout();
-	} );
-
-	it( 'can dismiss tax setup', async () => {
```

Comment:
Does this functionality no longer exist?

Reply:
These flows are included in the WC Admin test that we have replaced it with.